pkg/apiserver/experiment/experiment.go

```diff
@@ -81,8 +84,10 @@ func Register(r *gin.RouterGroup, s *Service) {
 	// TODO: add more api handlers
 	endpoint.GET("", s.listExperiments)
 	endpoint.POST("/new", s.createExperiment)
-	endpoint.GET("/detail/:namespace/:name", s.getExperimentDetail)
-	endpoint.DELETE("/:kind/:namespace/:name", s.deleteExperiment)
+	endpoint.GET("/detail/:kind/:namespace/:name", s.getExperimentDetail)
+	endpoint.DELETE("/delete/:kind/:namespace/:name", s.deleteExperiment)
```

Comment:
IMO `/delete/:kind/:namespace/:name` is not needed, do you mind changing this to `/:kind/:namespace/:name`? We are already using the `DELETE` verb here.